README.md

```diff
@@ -20,10 +20,12 @@ Run:
 ```bash
 cd typescript
 npm install
+
+sudo npm install -g gulp-cli
```

Comment:
This line should probably come first (maybe with a comment that `sudo` is just for Unix-based systems).